xbmc/GUIPassword.cpp

```diff
@@ -314,9 +313,8 @@ bool CGUIPassword::CheckLock(LockType btnType, const std::string& strPassword, i
       CProfilesManager::GetInstance().GetMasterProfile().getLockMode() == LOCK_MODE_EVERYONE || g_passwordManager.bMasterUser)
     return true;
 
-  int iVerifyPasswordResult = -1;
   std::string strHeading = g_localizeStrings.Get(iHeading);
-  iVerifyPasswordResult = VerifyPassword(btnType, strPassword, strHeading);
+  int iVerifyPasswordResult = VerifyPassword(btnType, strPassword, strHeading);
```

Comment:
Oh, right. I just had a brief look at the other calls checking for explicit -1. Sorry.